package.json

```diff
@@ -69,6 +69,7 @@
     "cozy-flags": "^2.7.2",
     "cozy-konnector-libs": "5.2.1",
     "cozy-logger": "^1.7.0",
+    "cozy-realtime": "^4.4.1",
```

Comment:
I think this commit 8e3ab2982e5e1f33cbde2e82640e619863d71f18 should be amend: it should only contains realtime stuff like with https://github.com/cozy/coachCO2/pull/174/commits/6a39906733eb197cceb84b1bc8d6bf2c5ad495fd
Currently, it includes D3 stuff that is modified in the next commit, a Backup component that is removed in the next commit, etc.